pkg/scaffold/v2/manager/kustomization.go

```diff
@@ -27,13 +27,15 @@ var _ input.File = &Kustomization{}
 // Kustomization scaffolds the Kustomization file in manager folder.
 type Kustomization struct {
 	input.Input
+	Images string 
```

Comment:
Sorry for not being clear enough.
The `images` fields I meant is in the generated `kustomization.yaml`, which should eventually looks like:
```yaml
namespace: project-v2-system
namePrefix: project-v2-

bases:
- ...
patches:
- ...

images:
- name: controller:latest
  newName: your/image
  newTag: yourTag
```

After thinking it more, I feel we don't have to change this file. We can change https://github.com/kubernetes-sigs/kubebuilder/blob/e960cb4216040355c39f85bf4a0febccfe983456/pkg/scaffold/v2/makefile.go#L101, which should be more easier for users to use.